app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeViewModel.kt

```diff
@@ -93,6 +88,35 @@ class ComposeViewModel
 
     private val mediaToDisposable = mutableMapOf<Long, Disposable>()
 
+
+    init {
+
+        Single.zip(api.getCustomEmojis(), api.getInstance(), BiFunction<List<Emoji>, Instance, InstanceEntity> { emojis, instance ->
```

Comment:
There should be `Singles` or something like that from Kotlin package to not write all these `BiFunction<...>`

Reply:
did not know this, nice